packages/chat-component/package.json

```diff
@@ -23,7 +23,8 @@
   "license": "MIT",
   "dependencies": {
     "dompurify": "^3.0.6",
-    "lit": "^2.8.0"
+    "lit": "^2.8.0",
+    "markdown-it": "^13.0.2"
```

Comment:
Why not use `marked` instead? It's more popular and way smaller/faster for frontend apps:

- https://bundlephobia.com/package/marked@9.1.5 => 10.5kb gzipped
- https://bundlephobia.com/package/markdown-it@13.0.2 => 35.5kb gzipped

Reply:
That's true. Updated the library.